src/casp/services/api/schemas/cell_operations.py

```diff
@@ -22,18 +22,46 @@ class DevDetailObject(BaseModel):
     mean_distance: float = Field(example=1791.372802734375)
 
 
-class MatchInfoDevDetails(MatchInfo):
+class AnnotationInfoCellTypeCountDevDetail(AnnotationInfoCellTypeCount):
     dataset_ids_with_counts: t.List[DevDetailObject]
 
 
-class QueryCell(BaseModel):
+class QueryCellAnnotationAbstract(BaseModel):
```

Comment:
Call it `QueryCellNeighborhoodAbstract`.

Annotation is a task we're trying to perform, but something we're getting from the query. Correct naming matters :)